deploy.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: admissionregistration.k8s.io/v1beta1
+kind: MutatingWebhookConfiguration
+metadata:
+  name: secrethub-mutating-webhook
+  labels:
+    app: secrethub-mutating-webhook
+    kind: mutator
+webhooks:
+- name: secrethub-mutating-webhook.default.svc.cluster.local
+  clientConfig:
+    url: "YOUR_CLOUD_FUNCTION_URL"
+    caBundle: ""
+  rules:
+  - operations: ["CREATE"]
+    apiGroups: [""]
+    apiVersions: ["v1"]
+    resources: ["pods"]
+  failurePolicy: Fail
```

Comment:
Let's do `deploy/gcloud-function/webhook.yml`.

Reply:
We could also move the deploy instructions `README` there if we want.

Reply:
And later we can add `deploy/k8s/config.yml` plus `deploy/k8s/webhook.yml`.

Reply:
Yes, sounds good :+1: